### DISCUSSION TITLE
有關ShokaX更新的方法

### DISCUSSION THREAD

**Discussion by XketsuroX** [theme-shoka-x/hexo-theme-shokaX#359] [Q&A] [answered]
請問一下有沒有什麽安全升級shokaX的方法，之前我試過直接用npm update， 結果config全被覆蓋了。

**Comment by D-Sketon** [comment, marked as answer]
你应该在 _config.shokax.yml 中编写你的配置，而非在node_modules中覆盖原有配置

**Reply by XketsuroX** [reply]
原來如此，另外想問一下有關圖片等asset是不是也可以丟到相同的path裏面

**Reply by D-Sketon** [reply]
参考以下路径
![{F5DD4F10-602B-42A7-8352-8B5DDA5B0B36}](https://github.com/user-attachments/assets/22f65dcc-d398-4dbb-a6b6-86ad8797061e)

**Reply by sugarcanecat** [reply]
请问如果想更新主题到最新版本，该如何操作呢？之前用的是即食罐头，即食罐头版本是0.4.14，目前版本是0.4.16
是只需要修改 shokax-can/package.json 中的"hexo-theme-shokax": "0.4.14"为0.4.16，然后再在shokax-can重新pnpm install就可以更新最新版本主题吗？
我查看我的即食罐头中 _config.shokax.yml没有任何内容
